buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -59,21 +63,34 @@ CustomInstallLib = patch_setuptools_command('install_lib')
     DEPENDS ${_outputs}
   )
 
+  # Specify the installation directory based on OS
+  if ( WIN32 )
```

Comment:
I don't like having this additional check here, but putting the logic in the `if (WIN32)` above did not work, as the function is not always called with the `EXECUTABLE` parameter, but we always need to set the correct installation directory.